bin/label-centers.py

```diff
@@ -1,14 +1,18 @@
 #!/usr/bin/env python3
+# -*- coding: utf-8 -*-
 
+# imports
 import os
 import sys
 import argparse
 import logging
 import nibabel as nb
 import numpy as np
 
-
 def label_volume_centers(label_volume):
+    """Crete a volume center and insert numpy to calculate it
+       while debugging it. """
+
```

Comment:
This comment makes no sense. The function aims to generate tuples (label, x, y, z)